### PR TITLE
Format CommandPipeline.__repr__

### PR DESCRIPTION
**Pull request by anki-code**
Continue the #3939

Additional news aren't required

Screenshot_20201029_183701

### CODE REVIEW

**Comment by carmenbianca** [comment]
Awesome @anki-code :+1:

Does Xonsh globally define somewhere what the preferred indentation level is? If yes, we might care to use that instead of a single space.

**Comment by anki-code** [comment]
Here I'm using two spaces.  I saw one in `xonsfig --json`. And all of these numbers less than 42 :(

**Comment by scopatz** [comment]
LGTM!

**Comment by scopatz** [comment]
There is [`$INDENT`](https://xon.sh/envvars.html#indent), but that is for input usually

File: xonsh/procs/pipelines.py [hunk 1]
```diff
@@ -175,8 +175,8 @@ def __init__(self, specs):
         self.proc = self.procs[-1]
 
     def __repr__(self):
-        s = self.__class__.__name__ + "("
-        s += ", ".join(a + "=" + repr(getattr(self, a)) for a in self.attrnames)
+        s = self.__class__.__name__ + "(\n  "
+        s += ",\n  ".join(a + "=" + repr(getattr(self, a)) for a in self.attrnames)
         s += ")"
```
Comment (scopatz) [inline xonsh/procs/pipelines.py:180]: I think the closing parenthese should be on its own line, so this should be `\n)`

**Comment by scopatz** [comment]
Thanks @anki-code!

**Pull request merged**